bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2ThingHandler.java

```diff
@@ -900,10 +900,7 @@ private void updateConnectivityState(Resource resource) {
             } else if (thing.getStatus() != ThingStatus.ONLINE) {
                 updateStatus(ThingStatus.ONLINE);
                 // issue REFRESH command to update all channels
-                Channel lastUpdateChannel = thing.getChannel(CHANNEL_2_LAST_UPDATED);
-                if (Objects.nonNull(lastUpdateChannel)) {
-                    handleCommand(lastUpdateChannel.getUID(), RefreshType.REFRESH);
-                }
+                handleCommand(new ChannelUID(thing.getUID(), "dummy"), RefreshType.REFRESH);
```

Comment:
It would be better if you could extract these lines to a separate method rather then "hacking" it this way with a dummy command:
https://github.com/openhab/openhab-addons/blob/74d55c60e18abfcf565fb3afa27051a0c6634e8c/bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2ThingHandler.java#L303-L316

Reply:
Ok. Shall we merge your refresh PR first, and then I can refactor that. Or we can combine both my code and yours in a single PR. ??

Reply:
I can include your changes in my PR, if that's okay with you. It would be the easiest for me since I would have to rebase and resolve conflicts anyway. The bugs are different, but the topic is the same, so release notes will also be clear.

Reply:
Done in #15736